data/json/items/tools.json

```diff
@@ -6112,7 +6112,56 @@
     "weight": 54000,
     "volume": 16,
     "bashing": 40,
-    "to_hit": -5
+    "to_hit": -5,
+    "qualities": [["ANVIL", 3]]
+  },
+  {
+    "id": "anvil",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "dark_gray",
+    "name": "anvil",
+    "description": "This is an enormously heavy block of oddly shaped steel with a chisel-like projection set into the corner.  It's used in most metalworking fabrication recipes.",
+    "price": 100000,
+    "material": "iron",
+    "flags" : ["DURABLE_MELEE"],
+    "weight": 54000,
+    "volume": 16,
+    "bashing": 40,
+    "to_hit": -5,
+    "qualities": [["ANVIL", 3]]
+  },
+  {
+    "id": "boulder_small",
+    "type": "TOOL",
+    "symbol": "o",
+    "color": "dark_gray",
+    "name": "small boulder",
+    "description": "This is a small block of stone.  It's sturdy enough to be used as an anvil.",
+    "price": 100,
+    "material": "stone",
+    "flags" : ["DURABLE_MELEE"],
+    "weight": 7885,
+    "volume": 9,
+    "bashing": 20,
+    "to_hit": -4,
+    "qualities": [["ANVIL", 2]]
+  },
+  {
+    "id": "boulder_medium",
```

Comment:
If those are only supposed to be pseudo items, they should have descriptions informing that you shouldn't be seeing them. And shouldn't have all the stats.
For example, the "fire" pseudo item - it can be used, but the item for fire can't actually be seen.

It's better to add new qualities as they are needed than to add placeholders, because placeholders aren't tested and someone might think that there is a working game mechanic that has to be preserved or something like that.